### PR TITLE
V1

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Python-API: Auth and Load](https://trello.com/c/4AqhEBPB/204-python-api-auth-and-load)

File: geoengine/lib.py [hunk 1]
```diff
@@ -1,23 +1,97 @@
...
+
+        data['start'] = gpd.pd.to_datetime(start)
+        data['end'] = gpd.pd.to_datetime(end)
+
+        return data
+
+    return geo_json_with_time_to_geopandas(data_response)
+
+
+def geopandas_by_workflow(workflow: str, bbox: Bbox) -> gpd.GeoDataFrame:
```
Comment (jdroenner) [inline geoengine/lib.py:90]: maybe "dataframe_by_workflow"

**Review by jdroenner — COMMENTED** [review]
i like it and have some suggestions. Additionally, i would like to have a get_result_descriptor method.

File: README.md [hunk 2]
```diff
@@ -2,6 +2,45 @@
...
+python3 -m pip install --upgrade build
+python3 -m build
+```
+
+## Try it out
+
+Start a python terminal and try it out:
+
+```python
+import geoengine as ge
```
[the remark this answers is not in the document]
Comment (ChristianBeilschmidt) [reply README.md:35]: in chat we agreed to ge

File: geoengine/lib.py [hunk 3]
```diff
@@ -1,23 +1,97 @@
...
+        headers=session.auth_header
+    ).json()
 
-    def get_features(self, workflow_id: Any, bbox: Any) -> gpd.GeoDataFrame:
-        pass
+    return WorkflowId(workflow_response)
 
 
-def initialize(server_url: str) -> Accessor:
+def geopandas_by_workflow_id(workflow_id: WorkflowId, bbox: Bbox, srs='EPSG:4326') -> gpd.GeoDataFrame:
```
[the remark this answers is not in the document]
Comment (ChristianBeilschmidt) [reply geoengine/lib.py]: 33c0257

File: geoengine/lib.py (hunk 1 above)
Comment (ChristianBeilschmidt) [reply geoengine/lib.py]: 33c0257

**Comment by ChristianBeilschmidt** [comment]
here is the result descriptor call
8deca95

**Pull request merged**